recreated the stdout/stderr snapshots with Rust 1.79.0-r1
by running `SNAPSHOTS=overwrite cargo test`

diff --git a/tests/cargo-upgrade/alt_registry/stderr.log b/tests/cargo-upgrade/alt_registry/stderr.log
index b285e5b..670ecfe 100644
--- a/tests/cargo-upgrade/alt_registry/stderr.log
+++ b/tests/cargo-upgrade/alt_registry/stderr.log
@@ -4,3 +4,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/cli_rust_version/stderr.log b/tests/cargo-upgrade/cli_rust_version/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/cli_rust_version/stderr.log
+++ b/tests/cargo-upgrade/cli_rust_version/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/exclude_dep/stderr.log b/tests/cargo-upgrade/exclude_dep/stderr.log
index dcb52a0..7ed0b2f 100644
--- a/tests/cargo-upgrade/exclude_dep/stderr.log
+++ b/tests/cargo-upgrade/exclude_dep/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/ignore_rust_version/stderr.log b/tests/cargo-upgrade/ignore_rust_version/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/ignore_rust_version/stderr.log
+++ b/tests/cargo-upgrade/ignore_rust_version/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/implicit_prerelease/stderr.log b/tests/cargo-upgrade/implicit_prerelease/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/implicit_prerelease/stderr.log
+++ b/tests/cargo-upgrade/implicit_prerelease/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/lockfile/stderr.log b/tests/cargo-upgrade/lockfile/stderr.log
index 4b8fa20..6c713e4 100644
--- a/tests/cargo-upgrade/lockfile/stderr.log
+++ b/tests/cargo-upgrade/lockfile/stderr.log
@@ -7,3 +7,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/optional_dep/stderr.log b/tests/cargo-upgrade/optional_dep/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/optional_dep/stderr.log
+++ b/tests/cargo-upgrade/optional_dep/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/package_rust_version/stderr.log b/tests/cargo-upgrade/package_rust_version/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/package_rust_version/stderr.log
+++ b/tests/cargo-upgrade/package_rust_version/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/pinned/stderr.log b/tests/cargo-upgrade/pinned/stderr.log
index 7604656..7903eb2 100644
--- a/tests/cargo-upgrade/pinned/stderr.log
+++ b/tests/cargo-upgrade/pinned/stderr.log
@@ -3,4 +3,5 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
 note: Re-run with `--pinned` to upgrade pinned version requirements
diff --git a/tests/cargo-upgrade/preserve_op/stderr.log b/tests/cargo-upgrade/preserve_op/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/preserve_op/stderr.log
+++ b/tests/cargo-upgrade/preserve_op/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/preserve_precision_major/stderr.log b/tests/cargo-upgrade/preserve_precision_major/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/preserve_precision_major/stderr.log
+++ b/tests/cargo-upgrade/preserve_precision_major/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/preserve_precision_minor/stderr.log b/tests/cargo-upgrade/preserve_precision_minor/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/preserve_precision_minor/stderr.log
+++ b/tests/cargo-upgrade/preserve_precision_minor/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/preserve_precision_patch/stderr.log b/tests/cargo-upgrade/preserve_precision_patch/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/preserve_precision_patch/stderr.log
+++ b/tests/cargo-upgrade/preserve_precision_patch/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/preserves_inline_table/stderr.log b/tests/cargo-upgrade/preserves_inline_table/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/preserves_inline_table/stderr.log
+++ b/tests/cargo-upgrade/preserves_inline_table/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/preserves_std_table/stderr.log b/tests/cargo-upgrade/preserves_std_table/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/preserves_std_table/stderr.log
+++ b/tests/cargo-upgrade/preserves_std_table/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/single_dep/stderr.log b/tests/cargo-upgrade/single_dep/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/single_dep/stderr.log
+++ b/tests/cargo-upgrade/single_dep/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/specified/stderr.log b/tests/cargo-upgrade/specified/stderr.log
index 2878527..dbdaebd 100644
--- a/tests/cargo-upgrade/specified/stderr.log
+++ b/tests/cargo-upgrade/specified/stderr.log
@@ -3,5 +3,6 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
 note: Re-run with `--verbose` to show more dependencies
   excluded: my-package2
diff --git a/tests/cargo-upgrade/upgrade_all/stderr.log b/tests/cargo-upgrade/upgrade_all/stderr.log
index 4b8fa20..6c713e4 100644
--- a/tests/cargo-upgrade/upgrade_all/stderr.log
+++ b/tests/cargo-upgrade/upgrade_all/stderr.log
@@ -7,3 +7,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/upgrade_everything/stderr.log b/tests/cargo-upgrade/upgrade_everything/stderr.log
index bcf2ce0..c0d3931 100644
--- a/tests/cargo-upgrade/upgrade_everything/stderr.log
+++ b/tests/cargo-upgrade/upgrade_everything/stderr.log
@@ -3,5 +3,6 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
 note: Re-run with `--verbose` to show more dependencies
   latest: serde
diff --git a/tests/cargo-upgrade/upgrade_renamed/stderr.log b/tests/cargo-upgrade/upgrade_renamed/stderr.log
index bb7bff6..59c1bfd 100644
--- a/tests/cargo-upgrade/upgrade_renamed/stderr.log
+++ b/tests/cargo-upgrade/upgrade_renamed/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/upgrade_verbose/stderr.log b/tests/cargo-upgrade/upgrade_verbose/stderr.log
index dcb52a0..7ed0b2f 100644
--- a/tests/cargo-upgrade/upgrade_verbose/stderr.log
+++ b/tests/cargo-upgrade/upgrade_verbose/stderr.log
@@ -3,3 +3,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/upgrade_workspace/stderr.log b/tests/cargo-upgrade/upgrade_workspace/stderr.log
index 4b8fa20..6c713e4 100644
--- a/tests/cargo-upgrade/upgrade_workspace/stderr.log
+++ b/tests/cargo-upgrade/upgrade_workspace/stderr.log
@@ -7,3 +7,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/virtual_manifest/stderr.log b/tests/cargo-upgrade/virtual_manifest/stderr.log
index 4b8fa20..6c713e4 100644
--- a/tests/cargo-upgrade/virtual_manifest/stderr.log
+++ b/tests/cargo-upgrade/virtual_manifest/stderr.log
@@ -7,3 +7,4 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
diff --git a/tests/cargo-upgrade/workspace_inheritance/stderr.log b/tests/cargo-upgrade/workspace_inheritance/stderr.log
index 6214f4d..32a5001 100644
--- a/tests/cargo-upgrade/workspace_inheritance/stderr.log
+++ b/tests/cargo-upgrade/workspace_inheritance/stderr.log
@@ -7,5 +7,6 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
 note: Re-run with `--verbose` to show more dependencies
   local: three
diff --git a/tests/cargo-upgrade/workspace_member_cwd/stderr.log b/tests/cargo-upgrade/workspace_member_cwd/stderr.log
index 6f3d82d..6aacb78 100644
--- a/tests/cargo-upgrade/workspace_member_cwd/stderr.log
+++ b/tests/cargo-upgrade/workspace_member_cwd/stderr.log
@@ -7,5 +7,6 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
 note: Re-run with `--verbose` to show more dependencies
   excluded: three
diff --git a/tests/cargo-upgrade/workspace_member_manifest_path/stderr.log b/tests/cargo-upgrade/workspace_member_manifest_path/stderr.log
index 6f3d82d..6aacb78 100644
--- a/tests/cargo-upgrade/workspace_member_manifest_path/stderr.log
+++ b/tests/cargo-upgrade/workspace_member_manifest_path/stderr.log
@@ -7,5 +7,6 @@
    Upgrading recursive dependencies
 warning: `[ROOT]/home/.cargo/config` is deprecated in favor of `config.toml`
 note: if you need to support cargo 1.38 or earlier, you can symlink `config` to `config.toml`
+     Locking 0 packages to latest compatible versions
 note: Re-run with `--verbose` to show more dependencies
   excluded: three
